scripts/all-tests.sh

```diff
@@ -12,7 +12,7 @@ set -e
 
 # Violations thresholds for failing the build
 export PYLINT_THRESHOLD=4175
-export JSHINT_THRESHOLD=7550
+export ESLINT_THRESHOLD=64345
```

Comment:
I think of line, file or directory-specific rule exceptions as a way to get us to "zero violations" while acknowledging that the rewards of doing certain maintenance tasks (ie - converting all variables to use camelCase) are less than the costs of refactoring that code (ie - breaking all code that calls your refactored file because the API has changed).

But that's based on the premise of getting to zero reported violations at some point, which maybe we're just not going to attempt.

(there's also several _kinds_ of JS in the repo -- some is requireJS/AMD, some is commonJS, some is just magic global variable house of cards stuff -- and not all the rules can apply logically to all the code. Example: we currently have some rules turned off in the Gulp directory of the toolkit, because it can use `'use strict';` differently than our browser code because it runs in Node).